spec/integration/workflows/steps.js

```diff
@@ -12,11 +13,19 @@ afterEach(() => {
 
 Given('Workflow with name {string}', (name) => {
   click('Add Workflow Button');
-  select('Empty workflow', 'Workflow Type Dropdown');
+  select('Empty workflow', 'Base Workflow Dropdown');
   type(name, 'Workflow Name');
   click('Workflow Add Button');
 });
 
+Given('add workflow popup is open', () => {
+  click('Add Workflow Button');
+});
+
 Then('Workflow appeared with name {string}', (name) => {
   cy.get('.selected-workflow button.mak').contains(name);
 });
+
+Then('{string} should have {string} {string} style', (element, cssValue, cssProperty) => {
+  $(element).should("have.css", cssProperty, styleValueSelector(cssValue));
+});
```

Comment:
sorry I might have been wrong about: `{string} should have {string} {string} style` we could move it back to common since it is common rule